docs/manual.rst

```diff
@@ -1530,9 +1483,9 @@ boundaries.
 .. code-block:: pycon
 
   >>> a.union(b).boundary
-  <shapely.geometry.polygon.LinearRing object at 0x...>
+  <shapely.LineString LINESTRING (1.435 -0.435, 1.293 -0.471, 1.147 -0.493, 1 ...>
```

Comment:
Also in Shapely 1.7/1.8, the `boundary` already returned a LineString instead of LinearRing